README.adoc

```diff
@@ -418,3 +420,22 @@ In response you should get ID of created item, using that you can retrieve the w
 ----
 $ ./bin/wit-cli show workitem --id <ID> -H localhost:8080 --pp
 ----
+
+== Prod preview setup
+
+In order to setup wit to work against link:https://prod-preview.openshift.io/[prod preview] one needs to update `token.publickey` in config.yaml.
+
+NOTE: Public key can be obtained from https://sso.prod-preview.openshift.io/auth/realms/fabric8/
+
+Also, the following environment variables must be set:
+
+* `export F8_DEVELOPER_MODE_ENABLED=1`
+* `export F8_OPENSHIFT_TENANT_MASTERURL="https://console.free-int.openshift.com/"`
+* `export F8_KEYCLOAK_REALM="fabric8"`
```

Comment:
@ibuziuk yes, I am sure.

You can use prod-preview via these two alternatives:
-  F8_KEYCLOAK_REALM=fabric8-test, that will use a test KC realm. The default public token is the one required by fabric8-test realm.
- F8_KEYCLOAK_REALM=fabric8, then you need to change the public key as you mentioned.

F8_OPENSHIFT_TENANT_MASTERURL needs to be updated cause our default value points to https://tsrv.devshift.net:8443. So your value is correct.

why do you need F8_DEVELOPER_MODE_ENABLED=1 ?